### PR TITLE
completed some test cases

### PR DESCRIPTION
**Pull request by Linkster700**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
   get 'profile/edit'
   get 'profile/load'
+  get 'profile/create'
```
Comment (bammons) [inline config/routes.rb]: delete this route, we don't need it.

File: app/controllers/profile_controller.rb [hunk 2]
```diff
@@ -11,12 +11,12 @@ def edit
     @info = User.all
   end
 
-  def create
-    @info = User.new(post_params)
+  # def create
```
Comment (bammons) [inline app/controllers/profile_controller.rb]: delete this function, we don't need it

File: db/schema.rb [hunk 3]
```diff
@@ -13,6 +13,23 @@
 
 ActiveRecord::Schema.define(version: 20140224012254) do
 
+  create_table "change_requests", force: true do |t|
```
Comment (bammons) [inline db/schema.rb]: This need to be re migrated. It's the old schema

run this command:
<code>rake db:drop && rake db:create && rake db:migrate && rake db:schema:dump && rake db:test:prepare</code>

File: spec/controllers/profile_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+  describe "GET 'edit'" do
+    it 'returns http success' do
+      get 'edit'
+
+      response.should be_success
+    end
+  end
+
+  # describe "GET 'create'"do
```
Comment (bammons) [inline spec/controllers/profile_controller_spec.rb]: delete this test case

File: spec/features/create_curriculum_spec.rb [hunk 5]
```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+feature 'Edit profile' do
+  background do
+    create(:user)
+    sign_in_with 'bammons123', '12345678'
+  end
+
+  scenario'Make a Curriculum'do
```
Comment (bammons) [inline spec/features/create_curriculum_spec.rb]: space between ' and do

File: spec/features/create_curriculum_spec.rb [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    make_curriculum_with 'This', 'Is AWESOME'
+    expect(page).to have_content('Successfully created curriculum ')
```
Comment (bammons) [inline spec/features/create_curriculum_spec.rb]: remove space after curriculum

File: spec/spec_helper.rb [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV['RAILS_ENV'] ||= 'test'
-require 'coveralls'
-Coveralls.wear!('rails')
+# require 'coveralls'
```
Comment (bammons) [inline spec/spec_helper.rb]: You definitely need to uncomment these lines when push back up.

**Comment by bammons** [comment]
-1 for the most part it's good, you just need to go through and make those small fixes

File: config/routes.rb (hunk 1 above)
Comment (Linkster700) [inline config/routes.rb]: done

File: app/controllers/profile_controller.rb (hunk 2 above)
Comment (Linkster700) [inline app/controllers/profile_controller.rb]: done

File: db/schema.rb (hunk 3 above)
Comment (Linkster700) [inline db/schema.rb]: done

File: spec/controllers/profile_controller_spec.rb (hunk 4 above)
Comment (Linkster700) [inline spec/controllers/profile_controller_spec.rb]: done

File: spec/features/create_curriculum_spec.rb (hunk 5 above)
Comment (Linkster700) [inline spec/features/create_curriculum_spec.rb]: done

File: spec/features/create_curriculum_spec.rb (hunk 6 above)
Comment (Linkster700) [inline spec/features/create_curriculum_spec.rb]: done

File: spec/spec_helper.rb (hunk 7 above)
Comment (Linkster700) [inline spec/spec_helper.rb]: done

**Comment by bammons** [comment]
+1

**Comment by kdorer** [comment]
+1

**Pull request merged**